utils/README.md

```diff
@@ -2,15 +2,22 @@
 
 Development utils to get and convert data
 
-# Actions
+# Exporting
 
 ## DocumentPlan JSON -> Semantic Graph
 
 ```bash
-clojure -m utils.document-plan to-semantic-graph <document-plan>.json
+clojure -M:export semantic-graph <document-plan>.json
 ```
 
-Example:
+## Exporting document plans
+
+Print single document plan:
+```bash
+clojure -M:export plan [plan-name]
+```
+
+Save all documents to a default `../api/resources/document-plans` location:
```

Comment:
maybe we should include the option to change export directory?

Reply:
Yes, as a second optional parameter would be cool